google-cloud-spanner/src/main/java/com/google/cloud/spanner/MetricRegistryConstants.java

```diff
@@ -45,9 +45,13 @@
   static final String MAX_IN_USE_SESSIONS = "cloud.google.com/java/spanner/max_in_use_session";
   static final String MAX_ALLOWED_SESSIONS = "cloud.google.com/java/spanner/max_allowed_sessions";
   static final String IN_USE_SESSIONS = "cloud.google.com/java/spanner/in_use_sessions";
+  static final String SESSIONS_TIMEOUT = "cloud.google.com/java/spanner/sessions_timeout";
```

Comment:
Suggestion: Call this `GET_SESSION_TIMEOUT`, as it is not a timeout on (a) session(s), but a timeout to get a session.

Reply:
Sure, done in 75af67669efbf1eb01cd16118f80c1d170063807